include/clap/factory/draft/preset-discovery.h

```diff
@@ -185,28 +193,21 @@ typedef struct clap_preset_discovery_filetype {
 
 // Defines a place in which to search for presets
 typedef struct clap_preset_discovery_location {
-   uint32_t    flags; // see enum clap_preset_discovery_flags
-   const char *name;  // name of this location
-
-   // URI:
-   // - file:/// for pointing to a file or directory; directories are scanned recursively
-   //   eg: file:///home/abique/.u-he/Diva/Presets/Diva (on Linux)
-   //   eg: file:///C:/Users/abique/Documents/u-he/Diva.data/Presets/ (on Windows)
-   //
-   // - plugin:// for presets which are bundled within the plugin DSO.
-   //   In that case, the uri must be exactly `plugin://` and nothing more.
-   const char *uri;
+   uint32_t    flags;    // see enum clap_preset_discovery_flags
+   const char *name;     // name of this location
+   uint32_t    kind;     // See clap_preset_discovery_location_kind
+   const char *location; // Actual location in which to crawl presets
```

Comment:
So, any mandates on the use of slashes on Windows? Both regular slashes and backslashes should be handled by the host? Probably a good idea to document that regardless, even thought it's unlikely to cause problems unless people implement their own file APIs.

Reply:
I think if the OS functions allows '/' which I think they do, then '/' are allowed.

Reply:
I will elaborate on the location attr